libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -47,76 +48,215 @@ typedef struct MqttPublishInfo MQTTPublishInfo_t;
 struct MQTTPacketInfo;
 typedef struct MQTTPacketInfo MQTTPacketInfo_t;
 
+/**
+ * @brief Signature of the transport interface receive function.
+ *
+ * A function with this signature must be provided to the MQTT library to read
+ * data off the network.
+ *
+ * @param[in] context The context provided with this function.
```

Comment:
The context --> The MQTT context.

Reply:
This is actually the network context, such as socket (int).  Updated wording.